etc/profiles.yml

```diff
@@ -0,0 +1,19 @@
+# The valid profile names are:
+# "microsoft-azure", "google-cloud-platform", "amazon-web-services, "user_customized"
+
+user_customized:
+  totem.token: 5000
+  totem.consensus: 6000
+  totem.join: 60
+  totem.max_messages: 20
+  totem.token_retransmits_before_loss_const: 10
+  sbd.watchdog_timeout: 5
+  sbd.msgwait: 10
```

Comment:
Maybe move other hard-coded parameters here too, for example,

    secauth:    on
    crypto_hash:    sha1
    crypto_cipher:  aes256
    cluster_name:   %(clustername)s
    clear_node_high_bit: yes

Reply:
Not suggest since these are rarely changed

Reply:
The point to add them here is trying to define a "full" list of parameters. What's is concern me a little is we don't clearly provide the full list. Therefore users are blind on this.

I agree, often, only the subset is needed. 

For example, we can see some AWS users doesn't use crypto. Well, probably the majority stay with crypto.
https://docs.aws.amazon.com/sap/latest/sap-hana/sap-hana-on-aws-cluster-configuration.html

Reply:
Let's comment out sbd settings here, so that `sbd create` chooses the proper defaults itself, which will be especially sensible for s390(x).

Reply:
OK

Reply:
Instead of delete sbd in "default", I prefer to add "s390" profile explicitly to override the default values. This avoid the hard-coded value for s390 and users just not aware them. This fits well for the concept to load profiles based on the environment.

Reply:
> Instead of delete sbd in "default", I prefer to add "s390" profile explicitly to override the default values. This avoid the hard-coded value for s390 and users just not aware them. This fits well for the concept to load profiles based on the environment.

Profiles are basically configurable. Any settings here could be changed or even unset by users... It's more or less another abuse of the term "default" in here :-) I mean the term still could be used in the context of profiles, but settings here shouldn't be considered by us as the "real internal defaults".

Reply:
You are right. "default" is not intended to be the "real internal defaults". It's naming is tricky. It's not perfect, we do agree.  Initially, we give it name "user_customized". Later on we name it to "default" for users to read the execution log more intuitive, 

eg. Loading form "default" profile from /etc/crmsh/profiles.yml. 

This "default" profile indeed let users to customize their own need, particular for those on-premise environments which is our of our pre-defined profiles in the yaml file.

Reply:
> This avoid the hard-coded value for s390 and users just not aware them.

And I think this is something that should be documented better in sbd manpage. Despite how it should be like in the shipped profiles, code-wise crmsh should anyway be able to handle the case where the settings are truly defaulted here.

Reply:
BTW, sounds make sense to have the sbd values in "default" profile for the gernal use according to bsc#1122604. 

sbd msgwait timeout 30s
watchdog timeout 15s

Reply:
Yes, definitely sensible to list them if they are different from the internal defaults.